zebedee/data.go

```diff
@@ -139,6 +139,7 @@ type TimeseriesDescription struct {
 type HomepageContent struct {
 	Intro           Intro               `json:"intro"`
 	FeaturedContent []Featured          `json:"featuredContent"`
+	AroundONS 		[]Featured          `json:"aroundONS"`
```

Comment:
Question: Do we need to handle backward compatibility here? I.e. Zebeded does not have the model? I think it should be fine we should test the client against a Zebedee version that does not have this new field to make sure it doesn't break.
If it does we could make it optional

Reply:
I've given it a test locally with zebedee not having the model property and the client is functioning fine, the rest of the page displays ok.